src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -68,23 +61,14 @@ const Begrunnelse = ({ klage }: Props) => {
 
     useEffect(() => window.scrollTo(0, 0), []);
 
-    const createKlageUpdate = (klage: Klage): UpdateKlage => ({
-        id: klage.id,
-        tema: klage.tema,
-        ytelse: klage.ytelse,
-        saksnummer: klage.saksnummer,
-        fritekst,
-        vedtak: dateToVedtakText(chosenDateOption, chosenISODate)
-    });
-
     useEffect(() => {
-        const timeout = setTimeout(
-            () => updateKlage(createKlageUpdate(klage)).catch((error: CustomError) => setError(error.message)),
-            1000
-        ); // 1s - timeout til å kjøre funksjon om timeouten ikke blir nullstillt
+        const timeout = setTimeout(() => {
+            const klageUpdate = createKlageUpdate(klage, fritekst, vedtakType, vedtakDate);
```

Comment:
Yup, it's back 😛 
Flyttet `createKlageUpdate` ut av komponenten med ekstra argumenter i stedet.
Da ble linjen for lang, men det er mer effektivt og vi slipper å ha den som `useEffect` dependency.